charts/turing/Chart.yaml

```diff
@@ -22,4 +22,4 @@ maintainers:
 - email: caraml-dev@caraml.dev
   name: caraml-dev
 name: turing
-version: 0.2.31
+version: 0.2.32
```

Comment:
Why did you decide to bump up Turing's version?

Reply:
Whoops, they were due to some chart changes I tried to do on Turing previously but I've since cut those changes out; will be reverting this.